web-ui/src/main/resources/catalog/views/default/templates/home.html

```diff
@@ -44,23 +44,20 @@ <h1 data-translate="">getstarted</h1>
 
             An icon could be provided as a CSS class name or
             a pattern matching a URL resolving to an image.
-
-            Properties
-            <div data-flfilter="categories"
-               data-title=""
-               data-icon-path="../../images/category/{{['@name']}}.png">
             -->
           <div class="row">
-            <div data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
-              class="col-xs-6 col-sm-4 col-lg-3">
-              <a class="thumbnail gn-icon-bg" data-ng-href="#/search?topicCat={{facet['@name']}}">
-                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i><br/>
-                <span class="gn-icon-label">{{facet['@label']}}</span>
+            <div data-ng-repeat="(key, facet) in searchInfo.facet['orgNames'] | limitTo:12"
```

Comment:
Why dropping topicCat choice in the home page ? It looks to be a quite general case which will be a good defaults in many catalogs. orgNames in many cases (eg. small organisation), will only contains one value and does not sounds much useful. 

BTW, this should probably move to the JS/admin configuration ?

Also why are you using orgNames facet and searching on topicCat - #/search?topicCat=. the icon is also useless, we should have organisation logo in that case ?


FYI, some user would like INSPIRE themes here - ongoing discussion about having a set of icons for that.